modules/nf-neuro/betcrop/synthbet/main.nf

```diff
@@ -2,7 +2,7 @@ process BETCROP_SYNTHBET {
     tag "$meta.id"
     label 'process_single'
 
-    container "freesurfer/synthstrip:latest"
+container "freesurfer/freesurfer:7.4.1"
     containerOptions "--entrypoint ''"
```

Comment:
The freesurfer container does not require overwriting the entry point; you can remove the line. Also, the indentation is missing before the container line.